### DISCUSSION TITLE
Leon's working on some ideas for a QUEST Storyline.

### DISCUSSION THREAD

**Discussion by hanatyan128** [OPENSPHERE-Inc/quest-i18n-resource#23] [Ideas]
From twitter post
https://twitter.com/leonsaidwhat/status/1560100636028534785

**Comment by leoinker** [comment]
Here is the HackMD page where I am working on the story: https://hackmd.io/@leoinker/QuestStoryDraft

The Symbol theme has friendly "criminal" element: EG: Pirates and Syndicates.
I tried to build off that theme by using terms such as "Bribes" and "Accomplices" in this QUEST Storyline. I also introduced a new mechanic called the "Connivance Score" - A simple algorithm to showcase a persons activity on QUEST.

**Comment by hanatyan128** [comment]
> Since my English skills are not good enough to convey the detailed nuances, I will write this in Japanese and hope you can translate it using DeepL or something similar.

このアイデアはQUESTの世界観をかなり「ファンタジー」に寄せた物だと理解します。そうした事を考えるのは常にエキサイティングです。そして、私たちは過去にそれを検討したことがありますが、結果としてあまりにもそちらに寄せすぎない事を選択しました。

また、究極的に言えばこれは Symbol が掲げる XYM City 構想に寄せる（というか XYM City 構想の一部になる？）というアイデアだと思いますが、私は XYM City を推進する立場ではないため、今のところそのような振る舞いには慎重でありたい考えています。これはアイデンティティの領域に踏み込みます。

次に、アイデアの利点と欠点を検討しましょう。

利点は XYM City 構想に賛同する人々と、日常をファンタジーの世界の様な価値観にしたい人々を QUEST に引き付ける事でしょう。欠点はそれ以外の大多数の人々を拒絶するだろうという事です。

QUEST を説明するうえでファンタジーの世界観は有用ですが、やりすぎると実態と乖離が大きくなります。QUEST の実態はもっと現実的です。私たちは現実を説明する為にファンタジーをソフトに用いることを選択しています。

用語の問題点についても述べたいと思います。コンプライアンス的に「わいろ」や「共犯者」はお金を扱うサービスとしては極めて不適切で、社会に対して無用の誤解を与えるでしょう。ファンタジーであると前置きしたとしても私は抵抗を覚えます。
ゲームの世界観としてはありですが、ユーザーは現実の法律に束縛されます。それらを曖昧にしてユーザーを QUEST へ誘うことは挑戦的ですが誤解されるリスクがあります。

ファンタジーを説明に用いることは賛成ですが、現実世界と地続きである事がユーザーに理解されなければなりません。

**Reply by leoinker** [reply]
I am slow to reply as I am getting over an illness.

I first wrote with a XYM City theme because QUEST uses Symbol. Anyone using QUEST will need to be equally interested in Symbol. Anyone that is using QUEST to raise funds is also using the Symbol blockchain. The core currency of QUEST is XYM, so the users need to want XYM.

Anyone that is opposed to Symbol and XYM is unlikely to be interested in using QUEST. To convince people to use QUEST, they need to be convinced to use Symbol and XYM, too.
It is important to show the users why Symbol blockchain is used, and not just in a boring way, but with passion. If the creators of QUEST do not show passion for Symbol, it is less likely the users will either.

XYMCity is the world the Symbol community resides in. It's not a tangible or static thing. It can be however we imagine. Anyone who loves Symbol is a XYMCity resident.
You don't need to be in any official position to use XYM City name and ideas. XYMCity is whatever we - the community - make it!

I do understand your concerns about terms like "accomplice" and the overall story becoming too wordy and fantastical. I chose high fantasy as I thought that was the intention with the current theme. 

Maybe a better starting point is to take a step back and determine what route you want to take. 
Is QUEST a competitor to services such as GoFundMe? https://www.gofundme.com/
Or is QUEST a new and niche service with the Symbol community as the primary target user? 

Should the story be more of a classic? - Who the founders are, what their dreams are, how the QUEST idea started, how QUEST is improving upon existing services, what the next goals are, etc.
Or should the story be more about Symbol and why we love it?

Asking questions like this will help clarify what purpose QUEST serves to the Symbol community, the crypto community, and society as a whole.

Lastly, the "story" isn't intended to be blasted on a single webpage, but rather is intended to be a full compilation of information dispersed throughout the site. A storyline for a game isn't presented all at once, but the relevant information is presented as needed for the user to understand the next steps.
If using any sort of fantasy or gaming element in QUEST, it would be unappealing to have a broken storyline.

I will await your thoughts! 
Thanks :)

**Reply by hanatyan128** [reply]
Sorry for the late reply.

QUEST の運営方針に触れる内容なので、返信は慎重に行っております。

QUEST の立ち位置としては（それが実際に出来ているかどうかは別として）広く一般に提供する「プラットフォーム」であり、当初より Symbol/XYM・暗号資産等、特定のコミュニティに限定しないという考えであり、QUEST の説明として XYM City を積極的に盛り込むことはしません。また、（今のところ）Symbol を始めとする基盤技術の選択についてもQUESTは自由度を留保します。私たちはSymbol をあくまでもQUESTというサービスを実現させるために適切な基盤技術の一つとして採用しています。それは採用している他のミドルウェア・クラウド製品とも同様です。

「Symbol に興味を持たない人は QUEST に興味を持たない」という結論には至っていません。QUEST が実現する機能について世の中のニーズに当てはまらないのであれば、当てはまるように改造を試みるのみです。

これを踏まえて、より進歩的な考え方をするならば、QUESTプラットフォーム上で「コミュニティ機能」として XYM City を含むご提案いただいた様なストーリーを持つユーザー独自のコミュニティが展開できるのが良いかなと思いました。

今後の機能拡張の参考にさせていただければと思います。

**Reply by leoinker** [reply]
I got distracted with other projects, but I want to finish writing this sometime in October.
I can write a more simplified description. Is it ok to call Quest a "Bounty hunt platform" or "Fund raising platform" ?
Then describe it as "a way for people to raise money for their project, cause, or other purpose." ?

**Reply by hanatyan128** [reply]
QUEST is officially Tipping platform, but Bounty hunt platform would be nice.
"Fund raising platform" is kinda misleading.
QUEST isn't called a crowdfunding platform because performer earns as a contingent fee.
